airflow/www/templates/airflow/graph.html

```diff
@@ -282,13 +282,21 @@
               tt += "Operator: " + task.task_type + "<br>";
               tt += "Started: " + ti.start_date + "<br>";
               tt += "Ended: " + ti.end_date + "<br>";
-              tt += "Duration: " + ti.duration + "<br>";
+              tt += "Duration: " + secondsToString(ti.duration) + "<br>";
               tt += "State: " + ti.state + "<br>";
               return tt;
             });
         });
     }
 
+    function secondsToString(seconds) {
+        var numdays = Math.floor((seconds % 31536000) / 86400); 
```

Comment:
Thinking do we really need magnitude to "days"... Personally, I think "hours" would be enough.

Reply:
Removing it would mean that one day someone will say "oh my task runs for days and it says 75 hours, I wish it would say the number of days". So I think better have it than not...